### PR TITLE
Landing about

### PR DESCRIPTION
**Pull request by ildyria**
Fixes https://github.com/LycheeOrg/Lychee-Laravel/issues/87
Fixes https://github.com/LycheeOrg/Lychee-Laravel/issues/86

Add more pages than just the landing one with Markdown support.
Does not provide with an editor. Should be in another PR.

### CODE REVIEW

File: app/Configs.php [hunk 1]
```diff
@@ -35,6 +35,16 @@ class Configs extends Model
 		'medium_max_width',
 		'medium_max_height',
 		'medium_2x',
+		'landing_title',
+		'landing_background',
+		'landing_facebook',
+		'landing_facebook',
```
Comment (d7415) [inline app/Configs.php]: Duplicate

File: resources/views/includes/footer.blade.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div id="footer" class="animate animate-up">
+    <p id="home_copyright">All images on this website are subject to Copyright by {{ $infos['owner'] }} &copy; 2019</p>
```
Comment (d7415) [inline resources/views/includes/footer.blade.php]: Not sure about hard codingthe year here. Maybe just leave it out?

File: resources/views/includes/menu.blade.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div id="menu_wrap" class="animate animate-down">
+    <div id="menu">
+        {{--<div class="menu-menu-1-container">--}}
```
Comment (d7415) [inline resources/views/includes/menu.blade.php]: Are these comments? If so, do we want them?

File: resources/views/page.blade.php [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        @if($infos['twitter'] != '')
+            <a href="{{ $infos['twitter'] }}" class="socialicons" id="twitter" target="_blank"></a>
+        @endif
+        @if($infos['instagram'] != '')
+            <a href="{{ $infos['instagram'] }}" class="socialicons" id="instagram" target="_blank"></a>
+        @endif
+        @if($infos['youtube'] != '')
+            <a href="{{ $infos['youtube'] }}" class="socialicons" id="youtube" target="_blank"></a>
+        @endif
+        {{--<a href="https://www.facebook.com/ThomasHeatonPhotography?ref=hl" class="socialicons" id="facebook" target="_blank"></a>--}}
```
Comment (d7415) [inline resources/views/page.blade.php]: Same question re comments

File: app/Configs.php (hunk 1 above)
Comment (ildyria) [reply app/Configs.php]: oups ^^

File: resources/views/includes/footer.blade.php (hunk 2 above)
Comment (ildyria) [reply resources/views/includes/footer.blade.php]: https://www.copyrightlaws.com/copyright-symbol-notice-year/

**Pull request merged**